source/localizable/tutorial/acceptance-test.md

```diff
@@ -0,0 +1,140 @@
+});
+```
+
+These tests will fail, since Ember tests will always fail without assertions.
+Since we have an idea of what we want our application to look like, we can also add some details to the tests.
+
+Ember acceptance tests provide test helpers to perform common tasks, such as visiting routes, filling in fields, clicking on elements, and waiting for pages to render.
+
+To check that rentals are listed, we'll first visit the index route and check that the results show 3 listings.
+
+```/tests/acceptance/list-rentals-test.js
+test('should list available rentals.', function (assert) {
+  visit('/');
+  andThen(function () {
+    assert.equal(this.$('.results .listing').length, 3, "should see 3 listings");
+  });
+});
+```
+The test assumes that we show the list of rentals in an element with class `results`, and that each rental will have an element with class `listing`.
+
+The [`visit`](http://emberjs.com/api/classes/Ember.Test.html#method_visit) helper loads the route specified for the given URL.
+
+The [`andThen`](../../testing/acceptance/#toc_wait-helpers) helper waits for all previously called test helpers to complete before executing the function you provide it.
+In this case we need to wait for the page to load after `visit`, so that we can assert that the listings are displayed.
+
+For the next two tests, we want to verify that clicking the about and contact page links successfully load the proper URLs.
+We'll use the [`click`](http://emberjs.com/api/classes/Ember.Test.html#method_click) helper to simulate a user clicking these links.
+After the new screen loads, we just verify that the new URL matches our expectation using the [`currentUrl`](http://emberjs.com/api/classes/Ember.Test.html#method_currentURL) helper.
+
+```/tests/acceptance/list-rentals-test.js
+test('should link to about page', function (assert) {
+  visit('/');
+  click('a:contains("About")');
+  andThen(function () {
+    assert.equal(currentURL(), '/about', "should navigate to about");
+  });
+});
+
+test('should link to contacts page', function (assert) {
+  visit('/');
+  click('a:contains("Contact")');
+  andThen(function () {
+    assert.equal(currentURL(), '/contact', "should navigate to contact");
+  });
+});
+```
+Note that we can call 2 [asynchronous test helpers](../../testing/acceptance/#toc_asynchronous-helpers) in a row without needing to use `andThen` or a promise.
+This is because each asynchronous test helper is made to wait until other test helpers are complete.
+
+Finally, we'll test that we can filter the list down according to a city search criteria.
+We anticipate having an input field in a container with a class of `list-filter`.
+We will fill out "Seattle" as the search criteria in that field, and send a key up event to trigger our filtering action.
+Since we control our data, we know that there is only one rental with a city of "Seattle", we assert that the number of listings is one, and that its location is "Seattle"
```

Comment:
with a city of "Seattle", **so** we assert .. is one (delete comma) and that its